### DISCUSSION TITLE
Questions about the adapter tests

### DISCUSSION THREAD

**Discussion by rluvaton** [axios/axios#5884] [Q&A]
### Describe the issue

I'm exploring the idea of adding undici support but want to know if the tests are running for each adapter?

I couldn't find if it does and just wanna make sure that in case I will add support, are the spec tests for example cover it automatically

**Comment by niksy** [comment]
@rluvaton did you manage to create adapter for Undici?

**Comment by jasonsaayman** [comment]
We are working on support for Undici
